src/main/groovy/com/palantir/gradle/shadowjar/ShadowJarPlugin.java

```diff
@@ -47,7 +47,9 @@ public class ShadowJarPlugin implements Plugin<Project> {
             groupOf("commons-logging"),
             groupOf("log4j"),
             groupOf("org.apache.logging.log4j"),
-            groupOf("com.palantir.logsafe.logger"),
+            groupOf("com.palantir.safe-logging"),
```

Comment:
I _think_ we can get away with only shading the `safe-logging` module from `com.palantir.safe-logging` as the rest can safely be repackaged. The smaller we can keep the set of ignored coordinates, the safer it'll be.